public/css/home.css

```diff
@@ -631,3 +631,8 @@ footer {
   padding: 0;
   margin: 0;
 }
+
+
+
+
```

Comment:
There are multiple lines at the end of this file. Is there any reason why you have this?